core/primitives-core/src/types.rs

```diff
@@ -18,6 +18,9 @@ pub type Nonce = u64;
 pub type BlockHeight = u64;
 /// Height of the epoch.
 pub type EpochHeight = u64;
+/// Block hash.
+#[cfg(feature = "protocol_feature_vm_hash")]
```

Comment:
Does a type alias need a feature check? It seems that it adds more complexity than its worth.

Reply:
Fair point